src/group_theory/specific_groups/cyclic.lean

```diff
@@ -365,6 +366,39 @@ end⟩
 
 end cyclic
 
+section quotient_center
+
+open subgroup
+
+variables {G : Type*} {H : Type*} [group G] [group H]
+
+/-- A group is commutative if the quotient by the center is cyclic.
+  Also see `comm_group_of_cycle_center_quotient` for the `comm_group` instance -/
+lemma commutative_of_cyclic_center_quotient [is_cyclic H] (f : G →* H)
+  (hf : f.ker ≤ center G) (a b : G) : a * b = b * a :=
+let ⟨⟨x, y, hxy⟩, hx⟩ := is_cyclic.exists_generator f.range in
```

Comment:
Maybe type-ascript `hx`/`hxy`? This is very hard to read without loading up Lean itself